experimental/packages/otlp-exporter-base/src/platform/browser/OTLPExporterBrowserBase.ts

```diff
@@ -30,6 +30,7 @@ export abstract class OTLPExporterBrowserBase<
   ServiceRequest,
 > extends OTLPExporterBase<OTLPExporterConfigBase, ExportItem, ServiceRequest> {
   protected _headers: Record<string, string>;
+  private _getHeaders?: () => Record<string, string>;
```

Comment:
nit for future readers, it's a little confusing why we have both `_headers` and `_getHeaders` and how they are related. Consider giving a name like `_getDynamicHeaders()` or `_getPerCallHeaders()`

Reply:
Thx - I've gone for changing to  `_getDynamicHeaders`